test/functional/services/lib/web_element_wrapper/web_element_wrapper.ts

```diff
@@ -105,25 +106,34 @@ export class WebElementWrapper {
 
   private async retryCall<T>(
     fn: (wrapper: this) => T | Promise<T>,
-    attemptsRemaining: number = RETRY_CLICK_MAX_ATTEMPTS
+    attemptsRemaining: number = RETRY_MAX_ATTEMPTS
   ): Promise<T> {
     try {
       return await fn(this);
     } catch (err) {
-      if (
-        !RETRY_CLICK_RETRY_ON_ERRORS.includes(err.name) ||
-        this.locator === null ||
-        attemptsRemaining === 0
-      ) {
+      if (!RETRY_ON_ERRORS.includes(err.name) || this.locator === null || attemptsRemaining === 0) {
         throw err;
       }
 
-      this.logger.warning(`WebElementWrapper.${fn.name}: ${err.message}`);
+      /*
+        https://bugs.chromium.org/p/chromedriver/issues/detail?id=4440
+        Starting with v113 driver occasionally returns WebDriverError instead of StaleElementReferenceError
+      */
+      if (err.message.includes('"code":-32000,"message":"No node with given id found"')) {
+        // WebDriverError: unknown error: unhandled inspector error: {"code":-32000,"message":"No node with given id found"}
+        this.logger.debug(`Chromedriver issue #4440, WebElementWrapper.${fn.name}: ${err}`);
+      } else {
+        this.logger.warning(`WebElementWrapper.${fn.name}: ${err}`);
+      }
+
       this.logger.debug(
-        `finding element '${this.locator.toString()}' again, ${attemptsRemaining - 1} attempts left`
+        `Searching again for the element '${this.locator?.toString()}', ${
+          attemptsRemaining - 1
+        } attempts left`
       );
 
       await setTimeoutAsync(200);
+      // WebElement reference id will be updated if element is no longer in DOM (StaleElementReferenceError)
       this._webElement = await this.driver.findElement(this.locator);
```

Comment:
For the future: WebDriver does not create a new reference id for each `findElement` call, but only for unique locators and if old reference (element is no longer in DOM) is invalid.
So potentially there is no need to reassign `this._webElement` for `ElementClickInterceptedError` and `ElementNotInteractableError` errors , its referenceId won't be changed. It might save run time on suites level, but would give it a try in separate PR.